+ source /root/mugen/libs/locallibs/common_lib.sh
++ python3 --version
++ '[' 0 -eq 0 ']'
++ source /root/mugen/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start to prepare the test environment.'
+ message='Start to prepare the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to prepare the test environment.'
Fri Jul 21 04:00:23 2023 - INFO  - Start to prepare the test environment.
+ DNF_INSTALL 'freeradius freeradius-utils'
+ pkgs='freeradius freeradius-utils'
+ node=1
+ '[' -z '' ']'
+ tmpfile=
++ python3 /root/mugen/libs/locallibs/rpm_manage.py install --pkgs 'freeradius freeradius-utils' --node 1 --tempfile ''
Fri Jul 21 04:00:23 2023 - INFO  - pkgs:(freeradius freeradius-utils) is already installed
+ tmpfile2=
+ '[' -z '' ']'
+ tmpfile=
+ echo 'steve  Cleartext-Password := "testing"
       Service-Type = Framed-User,
       Framed-Protocol = PPP,
       Framed-IP-Address = 172.16.3.33,
       Framed-IP-Netmask = 255.255.255.0,
       Framed-Routing = Broadcast-Listen,
       Framed-Filter-Id = "std.ppp",
       Framed-MTU = 1500,
       Framed-Compression = Van-Jacobsen-TCP-IP
    '
oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh: line 34: /etc/raddb/users: No such file or directory
+ systemctl start radiusd
Job for radiusd.service failed because the control process exited with error code.
See "systemctl status radiusd.service" and "journalctl -xeu radiusd.service" for details.
+ SLEEP_WAIT 2
+ wait_time=2
+ cmd=
+ mode=1
+ python3 /root/mugen/libs/locallibs/sleep_wait.py --time 2 --cmd '' --mode 1
+ LOG_INFO 'End to prepare the test environment.'
+ message='End to prepare the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to prepare the test environment.'
Fri Jul 21 04:00:25 2023 - INFO  - End to prepare the test environment.
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start to run test.'
+ message='Start to run test.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to run test.'
Fri Jul 21 04:00:25 2023 - INFO  - Start to run test.
+ radtest -d /etc/raddb/ steve testing localhost 0 testing123
+ grep Access-Accept
/usr/bin/radtest: line 115: hostname: command not found
(0) Error parsing "stdin": Failed to get value
+ CHECK_RESULT 1 0 0 'radtest -d execution failed.'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='radtest -d execution failed.'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'radtest -d execution failed.'
+ LOG_ERROR 'radtest -d execution failed.'
+ message='radtest -d execution failed.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'radtest -d execution failed.'
Fri Jul 21 04:00:25 2023 - ERROR - radtest -d execution failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 45'
+ message='oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 45'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 45'
Fri Jul 21 04:00:26 2023 - ERROR - oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 45
+ '[' 0 -eq 1 ']'
+ return 0
+ radtest -t pap steve testing localhost 0 testing123
+ grep Access-Accept
/usr/bin/radtest: line 115: hostname: command not found
(0) Error parsing "stdin": Failed to get value
+ CHECK_RESULT 1 0 0 'radtest -t execution failed.'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='radtest -t execution failed.'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'radtest -t execution failed.'
+ LOG_ERROR 'radtest -t execution failed.'
+ message='radtest -t execution failed.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'radtest -t execution failed.'
Fri Jul 21 04:00:26 2023 - ERROR - radtest -t execution failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 47'
+ message='oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 47'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 47'
Fri Jul 21 04:00:26 2023 - ERROR - oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 47
+ '[' 0 -eq 1 ']'
+ return 0
+ radtest -P udp steve testing localhost 0 testing123
+ grep Access-Accept
/usr/bin/radtest: line 115: hostname: command not found
(0) Error parsing "stdin": Failed to get value
+ CHECK_RESULT 1 0 0 'radtest -P execution failed.'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='radtest -P execution failed.'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'radtest -P execution failed.'
+ LOG_ERROR 'radtest -P execution failed.'
+ message='radtest -P execution failed.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'radtest -P execution failed.'
Fri Jul 21 04:00:26 2023 - ERROR - radtest -P execution failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 49'
+ message='oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 49'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 49'
Fri Jul 21 04:00:26 2023 - ERROR - oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 49
+ '[' 0 -eq 1 ']'
+ return 0
+ radtest -x steve testing localhost 0 testing123
+ grep Access-Accept
/usr/bin/radtest: line 115: hostname: command not found
(0) Error parsing "stdin": Failed to get value
+ CHECK_RESULT 1 0 0 'radtest -x execution failed.'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='radtest -x execution failed.'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'radtest -x execution failed.'
+ LOG_ERROR 'radtest -x execution failed.'
+ message='radtest -x execution failed.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'radtest -x execution failed.'
Fri Jul 21 04:00:26 2023 - ERROR - radtest -x execution failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 51'
+ message='oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 51'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 51'
Fri Jul 21 04:00:26 2023 - ERROR - oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 51
+ '[' 0 -eq 1 ']'
+ return 0
+ radtest -4 steve testing localhost 0 testing123
+ grep Access-Accept
/usr/bin/radtest: line 115: hostname: command not found
(0) Error parsing "stdin": Failed to get value
+ CHECK_RESULT 1 0 0 'radtest -4 execution failed.'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='radtest -4 execution failed.'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'radtest -4 execution failed.'
+ LOG_ERROR 'radtest -4 execution failed.'
+ message='radtest -4 execution failed.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'radtest -4 execution failed.'
Fri Jul 21 04:00:26 2023 - ERROR - radtest -4 execution failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 53'
+ message='oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 53'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 53'
Fri Jul 21 04:00:26 2023 - ERROR - oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 53
+ '[' 0 -eq 1 ']'
+ return 0
+ radtest -6 steve testing localhost 0 testing123
+ grep '\['
/usr/bin/radtest: line 115: hostname: command not found
(0) Error parsing "stdin": Failed to get value
+ CHECK_RESULT 1 0 0 'radtest -6 execution failed.'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='radtest -6 execution failed.'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'radtest -6 execution failed.'
+ LOG_ERROR 'radtest -6 execution failed.'
+ message='radtest -6 execution failed.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'radtest -6 execution failed.'
Fri Jul 21 04:00:26 2023 - ERROR - radtest -6 execution failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 55'
+ message='oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 55'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 55'
Fri Jul 21 04:00:26 2023 - ERROR - oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 55
+ '[' 0 -eq 1 ']'
+ return 0
+ systemctl stop radiusd
+ touch /var/log/radius/radutmp
touch: cannot touch '/var/log/radius/radutmp': No such file or directory
+ radwho -c
+ grep Name
radwho: Error reading or parsing radiusd.conf
+ CHECK_RESULT 1 0 0 'radwho -c execution failed.'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='radwho -c execution failed.'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'radwho -c execution failed.'
+ LOG_ERROR 'radwho -c execution failed.'
+ message='radwho -c execution failed.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'radwho -c execution failed.'
Fri Jul 21 04:00:26 2023 - ERROR - radwho -c execution failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 59'
+ message='oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 59'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 59'
Fri Jul 21 04:00:26 2023 - ERROR - oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 59
+ '[' 0 -eq 1 ']'
+ return 0
+ radwho -d /etc/raddb/
+ grep Name
radwho: Error reading or parsing radiusd.conf
+ CHECK_RESULT 1 0 0 'radwho -d execution failed.'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='radwho -d execution failed.'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'radwho -d execution failed.'
+ LOG_ERROR 'radwho -d execution failed.'
+ message='radwho -d execution failed.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'radwho -d execution failed.'
Fri Jul 21 04:00:26 2023 - ERROR - radwho -d execution failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 61'
+ message='oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 61'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 61'
Fri Jul 21 04:00:26 2023 - ERROR - oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 61
+ '[' 0 -eq 1 ']'
+ return 0
+ radwho -F /var/log/radius/radutmp
+ grep Name
radwho: Error reading /var/log/radius/radutmp: No such file or directory
+ CHECK_RESULT 1 0 0 'radwho -F execution failed.'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='radwho -F execution failed.'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'radwho -F execution failed.'
+ LOG_ERROR 'radwho -F execution failed.'
+ message='radwho -F execution failed.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'radwho -F execution failed.'
Fri Jul 21 04:00:26 2023 - ERROR - radwho -F execution failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 63'
+ message='oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 63'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 63'
Fri Jul 21 04:00:26 2023 - ERROR - oe_test_freeradius_freeradius-utils_radtestAndRadwho.sh line 63
+ '[' 0 -eq 1 ']'
+ return 0
+ LOG_INFO 'End to run test.'
+ message='End to run test.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to run test.'
Fri Jul 21 04:00:26 2023 - INFO  - End to run test.
+ CASE_RESULT 0
+ case_re=0
+ test -z 9
+ test 9 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Fri Jul 21 04:00:26 2023 - ERROR - failed to execute the case.
+ exit 9
+ post_test
+ LOG_INFO 'Start to restore the test environment.'
+ message='Start to restore the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to restore the test environment.'
Fri Jul 21 04:00:26 2023 - INFO  - Start to restore the test environment.
+ DNF_REMOVE
+ node=1
+ pkg_list=
+ mode=0
+ [[ -z '' ]]
+ [[ -z '' ]]
+ LOG_WARN 'no thing to do.'
+ message='no thing to do.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level warn --message 'no thing to do.'
Fri Jul 21 04:00:26 2023 - WARN  - no thing to do.
+ return 0
+ rm -rf /etc/raddb
+ rm -rf /var/log/radius
+ LOG_INFO 'End to restore the test environment.'
+ message='End to restore the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to restore the test environment.'
Fri Jul 21 04:00:26 2023 - INFO  - End to restore the test environment.
